src/main/java/ee/carlrobert/codegpt/util/FileUtils.java

```diff
@@ -107,4 +110,104 @@ private static Optional<Map.Entry<String, String>> findFirstExtension(
         .findFirst()
         .map(it -> Map.entry(it.getName(), it.getExtensions().get(0)));
   }
+
+  public static CharSequence getName(@NotNull String fileName, @Nullable String defaultValue) {
+    int index = StringUtilRt.lastIndexOf(fileName, '.', 0, fileName.length());
+    if (index < 0) {
+        return defaultValue;
+    }
+    return fileName.subSequence(0, index);
+  }
+
+  public static void replaceContentInFile(String from, String to, String input, String replace, int startLine, int endLine) {
+    if (input.indexOf("\r\n") > 0) {
```

Comment:
Let's use apache commons util function to normalize the string:
https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#replaceEach-java.lang.String-java.lang.String:A-java.lang.String:A-